packages/job-worker/src/playout/timeline/rundown.ts

```diff
@@ -146,7 +146,8 @@ export function buildTimelineObjsForRundown(
 			// If there is a valid autonext out of the current part, then calculate the duration
 			currentPartEnable.duration =
 				partInstancesInfo.current.partInstance.part.expectedDuration +
-				partInstancesInfo.current.calculatedTimings.toPartDelay
+				partInstancesInfo.current.calculatedTimings.toPartDelay +
+				partInstancesInfo.current.calculatedTimings.toPartPostroll // autonext should have the postroll added to it to not confuse the timeline
```

Comment:
I'm not immediately convinced by this.

Consider the case where we have a part playing a clip. The clip is 10s long, so will freeze after 10s. The expectedDuration of the part matches this.  
In this part is a graphic with a postRoll of 1s.

This change looks like the part will now play for 11s before the autonext happens, resulting in the clip freezing for 1s at the end while the graphic animates out.

Reply:
As far as I understand that doesn't happen. The part group would be 11 seconds long, but the piece enable for the clip would still be 10 seconds long, and the next part would be offset by `-nextPartInfo.calculatedTimings.fromPartRemaining` (which should be 1 second because of the postroll) so it would start after 10 seconds still.